### ISSUE TITLE
name conflict

### ISSUE THREAD

**Issue by steveschow** [timwedde/py_midicsv#12]
you should rename the command line midicsv and csvmidi scripts that you included.  When I did the pip install, it overwrote my existing binaries from John Walker that were already located in /user/local/bin...  

Secondly, if you're going to overwrite, then the pip installer should prompt me to make sure that is what I really want to do

thirdly, you should try to make your version of midicsv and csvmidi use the same command line options as the original binary so that if for some reason it does get overwritten and maybe I would even prefer to use your version for the Latin decoding...but if I have other scripts elsewhere based on using the binary midicsv, I don't want to break all those scripts.

**Comment by timwedde** [comment]
That is indeed a good catch.
I'll try and mirror the original CLI interface as much as possible so things that rely on the binary version can have a drop-in replacement. Should be coming in the next few days.

I'd be hesitant to patch user confirmation prompts into the pip installation process since -as far as I'm aware- that is not normally supported or recommended. It's also not always the case that one can divine whether the original midicsv tools are installed on a given system or not, making this a very hit-or-miss kinda deal.

Instead, I would propose mirroring the CLI interface (within reason) and potentially renaming the executables to maybe have something like a `py` prefix, i.e. `pymidicsv` and `pycsvmidi`.

**Comment by steveschow** [comment]
I renamed yours to midicsv.py

**Comment by timwedde** [comment]
I adjusted the executable names to `midicsvpy` and `csvmidipy` respectively in 33546da84412ed75c06340ce91f487e45f068e99.
Personally I'm not a fan of keeping file extensions (or things that look like file extensions) in executables as that makes it look like a file and not like a command.

I added `py` as a suffix such that one can easily auto-complete from the original command name instead of having to go back to the start of the line to add `py` at the front.

This will ship with the next release, which will probably be a 2.0 version bump since we're kinda changing the interface here.
